src/frontend/components/SearchBar/SearchBar.js

```diff
@@ -0,0 +1,48 @@
+import React from "react";
+//import "../App.css";
+import "./SearchBar.css";
+
+// Start array of user objects
+const users = [
+  {
+    id: 1,
+    name: 'Anitha Dharneedharan ',
+    Language: 'JavaScript',
+    Location: 'Sutton'
+  },
+  {
+    id: 2,
+    name: 'Florin Dumitru',
+    Language: 'Python',
+    Location: 'Walthamstow'
+  },
+  {
+    id: 3,
+    name: 'Kainy Ryu',
+    Language: 'Java',
+    Location: 'Bethnal Green'
+  },
+  {
+    id: 4,
+    name: 'Kamila Lew',
+    Language: 'C++',
+    Location: 'Heathrow'
+  },
+  {
+    id: 5,
+    name: 'Selchuk Karakus',
+    Language: 'Ruby',
+    Location: 'Leytonstone'
+  }
+];
+
+// End of user objects
+
+export default function SearchBar() {
+  return (
+    <div className="Search">
+      <input className="Search-language-input" type="search" placeholder="search language"/>
```

Comment:
For readability purposes, it might be a better idea to have the attributes assigned to this `Input` element each on different lines?